.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: "2.7" }
+          - { ruby: "2.7", concurrency: async-websocket }
+          - { ruby: "3.0" }
+          - { ruby: "3.1" }
+          - { ruby: ruby-head, ignore: true }
+          - { ruby: jruby-head, ignore: true }
+    name: test (ruby=${{ matrix.entry.ruby }}, concurrency=${{ matrix.entry.concurrency || 'none' }})
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.entry.ruby }}
+      - name: Set Concurrency
```

Comment:
I copied this from the [slack client project](https://github.com/slack-ruby/slack-ruby-client/blob/master/.github/workflows/test.yml) as suggested. Can you elaborate on why we're using this concurrency assignment for the 2.7 branch only?

Reply:
This is something very specific to slack-ruby-client, remove it all from here.